client/app/core/session.service.js

```diff
@@ -83,99 +99,10 @@ export function SessionFactory($http, $sessionStorage, gettextCatalog, $window,
 
   function active() {
     // may not be current, but if we have one, we'll rely on API 401ing if it's not
-    return model.token;
-  }
-
-  function setRBAC(productFeatures) {
-    RBAC.set(productFeatures);
-
-    // FIXME move these to the RBAC service as well
-    $state.actionFeatures = setRBACForActions(productFeatures);
-    $state.navFeatures = setRBACForNavigation(productFeatures);
-  }
-
-  function setRBACForNavigation(productFeatures) {
-    var features = {
-      dashboard: {show: entitledForDashboard(productFeatures)},
-      services: {show: entitledForServices(productFeatures)},
-      orders: {show: entitledForServices(productFeatures)},
-      requests: {show: entitledForRequests(productFeatures)},
-      catalogs: {show: entitledForServiceCatalogs(productFeatures)},
-      dialogs: {show: entitledForService(productFeatures)},
-      administration: {show: entitledForService(productFeatures)},
-    };
-    model.navFeatures = features;
-
-    return model.navFeatures;
-  }
-
-  function setRBACForActions(productFeatures) {
-    var features = {
-      serviceView: {show: angular.isDefined(productFeatures.service_view)},
-      serviceEdit: {show: angular.isDefined(productFeatures.service_edit)},
-      serviceTag: {show: angular.isDefined(productFeatures.service_tag)},
-      serviceDelete: {show: angular.isDefined(productFeatures.service_delete)},
-      serviceReconfigure: {show: angular.isDefined(productFeatures.service_reconfigure)},
-      serviceRetireNow: {show: angular.isDefined(productFeatures.service_retire_now)},
-      serviceRetire: {show: angular.isDefined(productFeatures.service_retire)},
-      serviceOwnership: {show: angular.isDefined(productFeatures.service_ownership)},
-    };
-    model.actionFeatures = features;
-
-    return model.actionFeatures;
-  }
-
-  function entitledForServices(productFeatures) {
-    var serviceFeature = lodash.find(model.actionFeatures, function(o) {
-      return o.show === true;
-    });
-
-    return angular.isDefined(serviceFeature);
-  }
-
-  function isAnyActionAllowed(actions, productFeatures) {
-    for (var i = 0; i <= actions.length; i++) {
-      if (angular.isDefined(productFeatures[actions[i]])) {
-        return true;
-      }
+    if (model.token !== null) {
```

Comment:
`return angular.isObject(model.token)? model.token : false;`
will that work here? (trade five lines for one)